math/benches/field.rs

```diff
@@ -57,12 +54,31 @@ pub fn f128_extension_ops(c: &mut Criterion) {
     let mut group = c.benchmark_group("f128_quad");
 
     group.bench_function("mul", |bench| {
-        let x = rand_value::<QuadExtensionA<f128::BaseElement>>();
-        let y = rand_value::<QuadExtensionA<f128::BaseElement>>();
+        let x = rand_value::<<f128::BaseElement as StarkField>::QuadExtension>();
+        let y = rand_value::<<f128::BaseElement as StarkField>::QuadExtension>();
         bench.iter(|| black_box(x) * black_box(y))
     });
 }
 
+pub fn batch_inv(c: &mut Criterion) {
```

Comment:
We could add it for other fields too - but I wouldn't expect to yield substantially different results because `batch_inv()` implementation is independent of the field. What we are comparing here is how `batch_inv()` compared to plain `inv()`.